Source/WebCore/platform/graphics/skia/ShareableBitmapSkia.cpp

```diff
@@ -69,8 +69,8 @@ void ShareableBitmap::paint(GraphicsContext&, float /*scaleFactor*/, const IntPo
 
 RefPtr<Image> ShareableBitmap::createImage()
 {
-    notImplemented();
-    return nullptr;
+    auto image = SkImages::RasterFromData(m_configuration.imageInfo(), SkData::MakeWithCopy(data(), sizeInBytes()), bytesPerRow());
```

Comment:
I don't think Skia has any equivalent of `cairo_surface_set_user_data()` so this is yet another copy of data...